### DISCUSSION TITLE
Donation Alert in Twitch Streams

### DISCUSSION THREAD

**Discussion by shameforest** [opencollective/opencollective#5938] [Q&A]
Hello!

We are a collective who want to do live streams on Twitch and collect donations through it.
All Donations should trigger a donation alert but the money should also be listed transparently on opencollective.

Is there any possibility to achieve this?

Kind Regards
Drio

**Comment by Betree** [comment]
Hi @mindopera, sorry for the late reply. That depends on the fiscal host you choose, but basically only the total amounts would be available; at the moment it is not possible to integrate with Twitch in a way that would surface each individual transaction in your budget.
